konstructio/kubefirst-docs
# fix: Updating docs for mkcert

## Description
Adding mkcert details to install instructions for all clouds and updating current 2.7 version with these changes
Added these details for Azure for vNext
Fixes instances of kubefirst for capitalization (other branding updates) 

## Related Issue(s)
None found

## How to test
Validate that content appears where expected before this step https://kubefirst.konstruct.io/docs/aws/quick-start/install/cli#create-your-new-kubefirst-cluster for each cloud
